examples/compute_intensive/README.md

```diff
@@ -0,0 +1,40 @@
+## Compute intensive example
+The purpose of this example is to test both cost of compute time and performance of disk operations.
+
+### How to run
+See run_test
+
+``` sh
+rm -fr .ert
```

Comment:
Remember to remove this. We are soon introducing the possibility to remove experiments and that would be the correct approach if the experiment have already been ran...